wellregistry/registry/admin/bulk_upload.py

```diff
@@ -43,8 +43,14 @@ def _get_county_lookup(country, state, county_name):
         return None
     return qs[0]
 
-
-def _get_monitoring_location(row, user):
+def _validate_decimal(field_name, dec_value, row_index, warning_messages):
+    try:
+        return Decimal(dec_value)
+    except:
+        warning_messages.append((row_index, {field_name:"Invalid Value '" + dec_value + "'"}))
```

Comment:
I wonder if string format notation is better than concatenation.

Reply:
can change if you want, otherwise it works and is readable